mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/RabbitMQEventBusTest.java

```diff
@@ -181,6 +185,36 @@ void registerGroupShouldCreateRetryExchange() throws Exception {
     class ConcurrentTest implements EventBusConcurrentTestContract.MultiEventBusConcurrentContract,
         EventBusConcurrentTestContract.SingleEventBusConcurrentContract {
 
+        @Test
+        void dispatchShouldHandleLoadBulksGracefully() throws Exception {
+            EventBusTestFixture.MailboxListenerCountingSuccessfulExecution countingListener1 = newCountingListener();
+            EventBusTestFixture.MailboxListenerCountingSuccessfulExecution countingListener2 = newCountingListener();
+            EventBusTestFixture.MailboxListenerCountingSuccessfulExecution countingListener3 = newCountingListener();
+
+            eventBus().register(countingListener1, new EventBusTestFixture.GroupA());
+            eventBus().register(countingListener2, new EventBusTestFixture.GroupB());
+            eventBus().register(countingListener3, new EventBusTestFixture.GroupC());
+            int totalGlobalRegistrations = 3; // GroupA + GroupB + GroupC
+
+            int threadCount = 10;
+            int operationCount = 10000;
+            int totalDispatchOperations = threadCount * operationCount;
+            ConcurrentTestRunner.builder()
+                .operation((threadNumber, operationNumber) -> eventBus().dispatch(EVENT, NO_KEYS))
+                .threadCount(threadCount)
+                .operationCount(operationCount)
+                .runSuccessfullyWithin(Duration.ofMinutes(10));
```

Comment:
Do we still need 10 minutes ?

Reply:
It's a maximum timeout, no? IMO it is all right.

Reply:
if we know it should succeed more than one order of magnitude faster. 
I would prefer it to fail fast if it should be the case.

Reply:
It takes 1 minute on my machine, would 3 minutes is okay to you? To be safe while running that test on CI.